### DISCUSSION TITLE
Send-later dialog text is not visible in dark theme

### DISCUSSION THREAD

**Discussion by Enpheebled** [Extended-Thunder/send-later#381] [Ideas]
I tried the send-later extension in Thunderbird 78.13.0 in Linux Mint 20 (That's the latest version in the Ubuntu repositories).

I have to use a dark theme in Thunderbird and on my computer generally due to an eye condition that makes it difficult to look at bright backgrounds.

It seems that the send-later dialog window does not display correctly with dark backgrounds. Presumably some of the colours in the dialog are hard-coded and do not adjust to system themes. This makes it impossible to read the text in the send-later dialog.

Is there any chance of an update that could address this issue?

I have attached a screenshot to illustrate the problem
![send-later-colours-problem](https://user-images.githubusercontent.com/13029060/134095505-35d6ee4e-2663-46dd-9e5f-eb1f657449c2.jpg)
.

**Comment by jph-sendlater** [comment]
It's either a bug in Thunderbird, or GTK, or the combination they used in the debian builds. I also put up with that for months. I don't know what caused it, but I do know that it's fixed in the latest version of Thunderbird. The new ESR version of Thunderbird is 91. I don't know when Mint will move to the new version.

**Comment by Enpheebled** [comment]
Ok thanks for letting me know, and for responding so quickly. Let's hope Thunderbird 91 doesn't take too long to arrive in the repos.

**Comment by mbaierl** [comment]
This issue still exists in Thunderbird 102.1.0 on the Mac in dark mode.

![CleanShot 2022-08-02 at 10 47 04@2x](https://user-images.githubusercontent.com/72055/182333181-8ceebc5a-0228-498e-b31b-049b77e32d70.png)

**Comment by jikamens** [comment]
Is this still an issue with Thunderbird 115 and Send Later 10.1.1 (or newer?)?

**Comment by jikamens** [comment]
Closed for no response.
